.github/workflows/ci-fixed.yml

```
@@ -0,0 +1,179 @@
+        ./scripts/run-tests.sh
+      env:
+        NODE_ENV: test
+        DB_HOST: localhost
+        DB_PORT: 5432
+        DB_NAME: pachedu_test
+        DB_USER: test
+        DB_PASS: test
+        REDIS_URL: redis://localhost:6379
+
+    - name: Upload backend coverage
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)